{
    "error": "Can only call api with demo api key for approved inputs!",
    "trace": "Traceback (most recent call last):\n  File \"/var/task/time_series_analytics.py\", line 196, in lambda_handler\n    raise ValueError('Can only call api with demo api key for approved inputs!')\nValueError: Can only call api with demo api key for approved inputs!\n"
}